### ISSUE TITLE
install.sh shows error

### ISSUE THREAD

**Issue by Haardispro** [heisenburgh/pixarch#9]
![image](https://user-images.githubusercontent.com/62053026/213900768-047a95d3-4dd7-4976-9b76-19bafb3c0f14.png)

please fix as soon as possible

**Comment by heisenburgh** [comment]
did u do sudo pacman -S dialog?

**Comment by Haardispro** [comment]
wait let me try again

**Comment by Haardispro** [comment]
Works well after installing dialog.
Some tweaks left to do in the installation scripts
Stuff like starship and rofi-power-menu left to add.
